screens/HomeScreen.js

```diff
@@ -1,12 +1,67 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+import { View, Text, TouchableOpacity } from 'react-native';
+import Colors from '../styles/colors';
+import * as Typography from '../styles/typography';
+import * as Spacing from '../styles/spacing';
+import { Video } from 'expo-av';
 
+const poster = require('../assets/icon.png');
 export default function HomeScreen({ navigation }) {
-    return (
-        <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
-            <Text
-                onPress={() => alert('This is the "Home" screen.')}
-                style={{ fontSize: 26, fontWeight: 'bold' }}>Home Screen</Text>
-        </View>
-    );
-}
\ No newline at end of file
+  return (
+    <>
+      <Video
+        style={{ flex: 1, position: 'fixed' }}
```

Comment:
```css
 position: 'fixed'
```
is creating a problem, like sometimes crashing the expo app
![image](https://user-images.githubusercontent.com/44284877/193980229-0dca0f86-c168-47a1-98d6-9833d1af0876.png)

Reply:
done.